validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -184,7 +184,8 @@ def _received_from_identity(self, zmq_identity):
         self._last_message_times[zmq_identity] = time.time()
         connection_id = self._identity_to_connection_id(zmq_identity)
         if connection_id not in self._connections:
-            self._connections[connection_id] = ("ZMQ_Identity", zmq_identity)
+            self._connections[connection_id] = \
+                ("ZMQ_Identity", zmq_identity, None)
```

Comment:
Any way we could make this a named tuple?  The purpose of the values (especially with that last `None`) as the tuple grows.